### ISSUE TITLE
Dynamis-Beaucedine ~ general bugs

### ISSUE THREAD

**Issue by Treble011** [EdenServer/community#1801]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x] I have provided my client version in the details. (type /ver into your game window)

### Details

- Vanguard Eyes might be a _little_ too squishy. Killed 19 of them and they seemed to melt (using melee).
- Statues zone-wide have incorrect loot pools. They should only drop hundos based on the type of statue family they are. As they are, they appear to have all 3 types in their loot pool and can theoretically drop all 3 (though we haven't witnessed it).
- Statues zone-wide can and do drop 2 infinity cores at one time. They should only drop one if any.
- AF drop rate might be too high. I didn't get an exact count but we got _quite_ a few. Check against city drop rates.
- The NM Mildaunegeux seems excessively difficult. Perhaps review code just to verify he is where you want him. We can schedule a time to kill him again even though we don't need him if you would like to observe (we got our attestation thankfully).
- Excluding Mildaunegeux, the NM Heavymail Djidzbad was the most difficult monster in the zone. Probably overtuned, especially considering he was just a general NM out of a statue.
- The pukis lose hate on the kiter when Angra Mainyu teleports even if the alliance have not engaged / damaged him yet.

**Comment by Treble011** [comment]
We also didn't get our "Dynamis-Beaucedine Interloper" titles upon completion. How will anyone ever know we were victorious?

**Comment by Treble011** [comment]
It should also be noted that LD did Glacier a few days ago and their AF drop rates did not seem high. It might have been a combination of good moon and the amount of farming we did.

**Comment by beauxq** [comment]
Should Hydra's Avatar be immune to sleep?
They are, and I don't think they should be.

**Comment by Treble011** [comment]
They shouldn't be but I know I saw it stick a few times tonight too. It might just be severely resistant.

**Comment by Treble011** [comment]
They probably share resistance with their master, I had no problem on a fully merited bard but I don't on any hydra.

**Comment by Mozzy4Ever** [comment]
Going to add/update to this with recent info as well:

Hee Mida is apparently RNG and NIN, and will use both 2hrs at the same time? Can't find any data on this being a thing in era. Posted @ #2133  as well

AF drop rates were about what we expected considering how much we killed (though I'm starting to wonder if RDM relic exists now....)

Didn't have any issues with Heavymail Djidzbad @ G-10, so I presume he was tuned.

On the last bullet about pukis losing hate - Angra seems to be deleting it's enmity table after teleporting. Each time it teleported, it would stay at it's position and be white while we ran to it. 

Angra also would teleport rapidly. To the point that as a RDM, I could rarely get a Poison 2 off on it because in the time it took me to run to it and start casting, it would already teleport again. I'm guessing this is because it would be attacked by the carbuncle chasing it around so it would start it's waiting timer until next teleport. By the time we caught up to it it would be ready to teleport again.
